app/model/FacultyModel.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: Michal Struna
+ * Date: 24/11/2018
+ * Time: 16:19
+ */
+
+namespace App\Model;
+
+
+class FacultyModel extends BaseModel implements IFacultyModel, IDatabaseWrapper {
+
+    public function getAll(): array {
+        return $this->database->fetchAll('SELECT * FROM sem_fakulta');
+    }
+
+    public function getById(string $id) {
+        return $this->database->fetch('SELECT * FROM sem_fakulta WHERE id = ?', $id);
+    }
+
+    public function updateById(string $id, array $changes): void {
+        $this->database->query(
+            'UPDATE sem_fakulta SET nazev = ?, zkratka = ? where id = ?',
```

Comment:
Nevím, jestli cheme hodnoty dosazovat manuálně, nebo nějak rozparsovat to pole. Jestli chceš, klidně na to koukni.

Reply:
Muzem rucne... Muzes to dat aby se ti to tam vypsalo samo, ale nevim jak bychom zarucili poradi tech veci... (3 tecky by to mely z pole vyplyvnout do formy ze by to slo...)